_posts/2024-12-18-what-is-new-in-strimzi-0.45.0.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title:  "Video: What's new in Strimzi 0.45.0"
+date: 2024-12-18
+author: ppatierno
```

Comment:
```suggestion
author: paolo_patierno
```

Reply:
Wow! Thanks for catching this Jakub! Fixed!

Reply:
It was visible on the preview.